python/cuvs/cuvs/neighbors/cagra/cagra.pyx

```diff
@@ -219,6 +247,10 @@ cdef class IndexParams:
     def nn_descent_niter(self):
         return self.params.nn_descent_niter
 
+    @property
+    def refinement_rate(self):
+        return self.params.graph_build_params.refinement_rate
+
```

Comment:
maybe we should add getters for the ivf_pq_search_params/ivf_pq_build_params too ? 
```python
    @property
    def ivf_pq_search_params(self):
        return self.ivf_pq_search_params

    @property
    def ivf_pq_build_params(self):
        return self.ivf_pq_build_params
```